src/js/ConnectionService.js

```diff
@@ -1,150 +1,371 @@
 
 connectionModule = angular.module('ConnectionServiceModule', [])
 
-connectionModule.factory('connectionService',['$log', '$q', function($log, $q) {
+connectionModule.factory('connectionService',['$log', '$q', '$routeParams', function($log, $q, $routeParams) {
   
-  var connectionList = [];
-  
-  /**
-    * To initiate ssh connections to remote clusters.
-    *
-    */
-    
-  var getConnection = function(host) {
-    
-    // Check if the host exists in the conneciton list
-    if( connectionList.hasOwnProperty(host)) {
-      return connectionList[host];
-    } else {
-      return null;
-    }
-    
-  };
-  
-  
-  var commandSem = require('semaphore')(1);
-  
-  var runCommand = function(command) {
-    
-    var deferred = $q.defer();
-    
-    commandSem.take(function() {
-
-      // Run a command remotely
-      connectionList[0].exec(command, function(err, stream) {
-        
-        cumulData = "";
-        
-        if (err) {
-          $log.error("Error running command " + command + ": "+ err);
-          commandSem.leave();
-          deferred.reject("Error running command " + command + ": "+ err);
-          return;
-        }
-        
-        stream.on('data', function(data) {
-          
-          $log.debug("Got data: " + data);
-          cumulData += data;
-          
-        }).on('close', function(code, signal) {
-          
-          $log.debug('Stream :: close :: code: ' + code + ', signal: ' + signal);
-          commandSem.leave();
-          deferred.resolve(cumulData);
-          
-        });
-        
-
-      });
-      
+	var connectionList = {crane: null,
+							tusker: null,
+							sandhills: null,
+							glidein: null};
+	$log.debug(connectionList);
```

Comment:
Ya, the tabbing is throughout.  Please go through your files and remove all tabs!